src/devices/machine/tc009xlvc.cpp

```diff
@@ -14,186 +14,140 @@
 #include "emu.h"
 #include "machine/tc009xlvc.h"
 
+#include "emupal.h"
 #include "screen.h"
 
 
 DEFINE_DEVICE_TYPE(TC0091LVC, tc0091lvc_device, "tc009xlvc", "Taito TC0091LVC")
 
-
-READ8_MEMBER(tc0091lvc_device::tc0091lvc_paletteram_r)
+u8 tc0091lvc_device::rom_r(offs_t offset)
 {
-	return m_palette_ram[offset & 0x1ff];
+	return m_rom[offset & m_rom.mask()];
 }
 
-WRITE8_MEMBER(tc0091lvc_device::tc0091lvc_paletteram_w)
+void tc0091lvc_device::vram_w(offs_t offset, u8 data)
 {
-	m_palette_ram[offset & 0x1ff] = data;
-
+	/* TODO : offset 0x0000 - 0x3fff is not used?
+	if (offset < 0x4000)
+		return;
+	*/
+
```

Comment:
This just changes implementation, also again it's not declaring an extra address space.

Reply:
Moved into lw8 in map(0x14000, 0x1ffff).